tests/e2e/core-tests/specs/shopper/front-end-checkout-registered.test.js

```diff
@@ -0,0 +1,130 @@
+			// Sell to all countries
+			await expect(page).toSelect('#woocommerce_allowed_countries', 'Sell to all countries');
+			// Set currency to USD
+			await expect(page).toSelect('#woocommerce_currency', 'United States (US) dollar ($)');
+			// Tax calculation should have been enabled by another test - no-op
+			// Save
+			await settingsPageSaveChanges();
+
+			// Verify that settings have been saved
+			await Promise.all([
+				expect(page).toMatchElement('#message', {text: 'Your settings have been saved.'}),
+				expect(page).toMatchElement('select[name="woocommerce_default_country"]', {text: 'United States (US) — California'}),
+				expect(page).toMatchElement('#woocommerce_allowed_countries', {text: 'Sell to all countries'}),
+				expect(page).toMatchElement('#woocommerce_currency', {text: 'United States (US) dollar ($)'}),
+			]);
+
+			// Enable BACS payment method
+			await merchant.openSettings('checkout', 'bacs');
+			await setCheckbox('#woocommerce_bacs_enabled');
+			await settingsPageSaveChanges();
+
+			// Verify that settings have been saved
+			await verifyCheckboxIsSet('#woocommerce_bacs_enabled');
+
+			// Enable COD payment method
+			await merchant.openSettings('checkout', 'cod');
+			await setCheckbox('#woocommerce_cod_enabled');
+			await settingsPageSaveChanges();
+
+			// Verify that settings have been saved
+			await verifyCheckboxIsSet('#woocommerce_cod_enabled');
+
+			// Enable PayPal payment method
+			await merchant.openSettings('checkout', 'paypal');
+			await setCheckbox('#woocommerce_paypal_enabled');
+			await settingsPageSaveChanges();
+
+			// Verify that settings have been saved
+			await verifyCheckboxIsSet('#woocommerce_paypal_enabled');
+
+			await merchant.logout();
+		});
+
+		it('allows existing customer to place order', async () => {
+			await shopper.login();
+			await shopper.goToShop();
+			await shopper.addToCartFromShopPage(simpleProductName);
+			await shopper.goToCheckout();
+			await shopper.productIsInCheckout(simpleProductName, `1`, singleProductPrice, singleProductPrice);
+			await shopper.fillBillingDetails(config.get('addresses.customer.billing'));
+
+			await uiUnblocked();
+
+			await expect(page).toClick('.wc_payment_method label', {text: 'Cash on delivery'});
+			await expect(page).toMatchElement('.payment_method_cod', {text: 'Pay with cash upon delivery.'});
+			await uiUnblocked();
+			await shopper.placeOrder();
+
+			await expect(page).toMatch('Order received');
+
+			// Get order ID from the order received html element on the page
+			let orderReceivedHtmlElement = await page.$('.woocommerce-order-overview__order.order');
+			let orderReceivedText = await page.evaluate(element => element.textContent, orderReceivedHtmlElement);
+			return orderId = orderReceivedText.split(/(\s+)/)[6].toString();
+		});
+
+		it('store owner can confirm the order was received', async () => {
```

Comment:
Do we need to have the merchant check for the order in every test that creates an order? A couple alternatives for discussion:

- If so, this would be a good opportunity to turn the merchant order check into a function.
- The api could be used to verify that orders in additional tests were created.